src/config/database.ts

```diff
@@ -11,7 +11,7 @@ export const database = (configService: ConfigService): MysqlConnectionOptions =
     username: configService.get<string>('DATABASE_USERNAME'),
     password: configService.get<string>('DATABASE_PASSWORD'),
     entities: [__dirname + '/../**/*.entity{.ts,.js}'],
-    synchronize: configService.get<boolean>('SYNC_DATABASE'),
+    synchronize: configService.get<boolean>('SYNC_DATABASE') == true,
```

Comment:
Colocar o sincronize desse jeito não faz ele sempre ser true? Ou é equivalente ao 'SYNC_DATABASE' | true?

Reply:
Eu to comparando a variável de ambiente a true, ou "true" se o configService me retornar string. Se for, vai continuar true, senão é false. To fazendo isso porque eu não sei o que esse configService retorna, mesmo com o generics do get, não sei se ele transforma string "true" em true booleano

Reply:
Bizarro isso, o código anterior na lógica dá na mesma.